django_docker_engine/proxy.py

```diff
@@ -62,9 +62,14 @@ def _render(self, context):
         return template.render(context)
 
     def url_patterns(self):
+        if self.csrf_exempt:
+            proxy_view = csrf_exempt_decorator(self._proxy_view)
+        else:
+            proxy_view = self._proxy_view
+
         return [url(
             r'^(?P<container_name>[^/]*)/(?P<url>.*)$',
-            self._proxy_view
+            proxy_view
```

Comment:
I'd be tempted to use an inline if/else here (but I think I overuse inline if/else)